python_modules/libraries/dagster-k8s/dagster_k8s/executor.py

```diff
@@ -43,6 +44,14 @@
                 description="Limit on the number of pods that will run concurrently within the scope "
                 "of a Dagster run. Note that this limit is per run, not global.",
             ),
+            "send_args_via_cli": Field(
```

Comment:
you might be able to convince me we don't need this... we don't test version differences between run worker and step worker, and I don't think we've ever claimed to support it?

Reply:
yeah i don't think supporting back-compat there is essential at this stage